Github: Remove PR template

OrkunTokdemir opened the pull request:
Since we migrated to Github, we no longer need to use Gerrit.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/qt-labs/vscodeext?pullRequest=1) <br/>All committers have signed the CLA.

OrkunTokdemir committed:
Github: Update pull_request_template.md

Since we migrated to Github, we don't need to use Gerrit anymore.
This PR removes the Gerrit related information from the PR template.

jobor reviewed (approved):
Excellent first contribution!
